site/src/pages/CreateWorkspacePage/CreateWorkspacePage.tsx

```diff
@@ -81,11 +110,13 @@ const CreateWorkspacePage: FC = () => {
           defaultOwner={me}
           defaultBuildParameters={defaultBuildParameters}
           error={error}
-          template={template as Template}
+          template={template!}
```

Comment:
@BrunoQuaresma Still haven't had the chance to look through XState, so I have no idea here: is XState just not able to make more type-safe guarantees? I'm surprised that it requires all these `undefined` checks

Reply:
It is something they have to improve for sure. It is not 100% type-safe in v4 but looks like they are looking to improve it in v5.